bin/emacs_files_that_contain_string.bash

```diff
@@ -21,7 +21,7 @@ echo " "
 fi
 
 #Search string 
-search_string=RefineableTriangleMesh
+search_string=solve_eigenproblem
```

Comment:
As a note, these files do not necessarily need to be updated in the repository. There was a way to do this in svn that we never really used and I don't feel strongly enough about it to investigate whether a similar trick is possible in git.

Reply:
Good point

@MatthiasHeilManchester, see [here](https://stackoverflow.com/a/3320183) for info on how precisely to do this

**TL;DR:** Use 

```bash
git update-index --assume-unchanged bin/emacs_files_that_contain_string.bash
```

to tell Git to not track changes on that file as long as you want. When you want to make Git track it again, use

```bash
git update-index --no-assume-unchanged bin/emacs_files_that_contain_string.bash
```

You will encounter a Git error when you run `git pull` if you have modified that file. In this case, just undo the changes using 

```bash
git checkout bin/emacs_files_that_contain_string.bash
```

or just stash the changes, pull in the changes and unstash

Reply:
They should be updated/tracked in case anybody makes any improvements to them. I suppose it's more a question if minor edits (like this one, which I suppose could have been accommodated by passing that argument to the command line) should be committed; this is always the user's decision, to be taken on a case-by-case basis.